zproject/computed_settings.py

```diff
@@ -608,6 +608,8 @@
     DIRS=[
         # The main templates directory
         os.path.join(DEPLOY_ROOT, "templates"),
+        # The help documentation directory
+        os.path.join(DEPLOY_ROOT, "help"),
```

Comment:
Does this need to be here? I don't think those files are Jinja2 templates.

Reply:
Nope. Removed this change.

Now the help center docs are using the `pure_markdown` boolean and the absolute path so that in `render_markdown_path` they are rendered not by the jinja template engine.